From c1f79e5b7f2dc335983ba7df981b5d45170295fb Mon Sep 17 00:00:00 2001
From: Manoj Srivastava <srivasta@golden-gryphon.com>
Date: Wed, 14 May 2014 23:48:09 -0700
Subject: [PATCH 1/3] debcherry fixup patch

32318ec [master]: New upstream release
	 - no changes against upstream or conflicts
786d26c [master]:  New upstream release
	 - extra changes or conflicts
36e907c [topic--debian]: FTBFS due to binutils-gold
	 - extra changes or conflicts
1021e15 [master]: First phase of a ove to a new build system using dh.
	 - no changes against upstream or conflicts
23433b4 Added the debian submodule.
	 - extra changes or conflicts
b15c77f Reduce divergence from upstream, and move to sdl variant
	 - extra changes or conflicts
815d966 Set the maintainewr address to be a debian role address
	 - no changes against upstream or conflicts
a83a04d Fix typo in man page
	 - extra changes or conflicts
a3d690b Revert the -m32 flags; use linux32 instead at run-time
	 - extra changes or conflicts
79cab02 Build in 32bit mode
	 - extra changes or conflicts
de0507e Fix initialization of directory for loading/saving games.
	 - extra changes or conflicts
ae5a5b4 Really synch with upstream
	 - extra changes or conflicts
a2f1a5c Added a manual page
	 - extra changes or conflicts
18bd448 Changes to integrate tome into Debian
	 - extra changes or conflicts
---
 src/config.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/config.h b/src/config.h
index 7c1cf1d..c9f460e 100644
--- a/src/config.h
+++ b/src/config.h
@@ -89,7 +89,7 @@
 /*
  * OPTION: Include "ncurses.h" instead of "curses.h" in "main-gcu.c"
  */
-/* #define USE_NCURSES */
+#define USE_NCURSES
 
 
 /*
@@ -226,7 +226,7 @@
  * that file.
  */
 #ifndef DEFAULT_PATH
-# define DEFAULT_PATH "./lib/"
+# define DEFAULT_PATH "/var/games/tome"
 #endif
 
 
@@ -261,7 +261,7 @@
 /*
  * OPTION: Person to bother if something goes wrong.
  */
-#define MAINTAINER      "darkgod@t-o-m-e.net"
+#define MAINTAINER      "tome@packages.debian.org"
 
 
 /*
@@ -282,7 +282,7 @@
 /* ToME options: */
 
 /* Should the player know his / her starting life rate? */
-/* #define SHOW_LIFE_RATE */
+#define SHOW_LIFE_RATE
 
 /* Allow hordes of 'similar' monsters */
 #define MONSTER_HORDES
-- 
2.0.0.rc2

